### PR TITLE
The same of Maven

### PR DESCRIPTION
**Pull request by TecEliMoran**
NEW ENTRANCES:

-[marven-app]:
This folder contains the points 1, 2 and 3. 
the app asks you for your name and then answer hello and your name.
I also added the dependency "log4j" to the pom.xml archive and compiled after that.
(marven\* instead of maven is just a spelling mistake)

-[my-maven-webapp]: 
a simple maven web app.

-[pom.xml]:
I modified this archive to create a multimodule proyect but y don't seem to be working.
(also modified the "marven-app"'s and "my-maven-webapp"'s pom.xml folder as son module)
I found out that there is a wat to create a multimodule proyect in one line, but I don't know 
how to use it yet.
So, a .war file was created  my-maven-webapp/target/my-maven-webapp.war

### CODE REVIEW

File: Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.bootcamp.app;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (npogulanik) [inline Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java:10]: You don't have to extend a class here

File: Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ */
+public class AppTest 
+    extends TestCase
+{
+    /**
+     * Create the test case
+     *
+     * @param testName name of the test case
+     */
+    public AppTest( String testName )
```
Comment (npogulanik) [inline Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java:18]: Is not necessary to call a constructor

File: Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        super( testName );
+    }
+
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
```
Comment (npogulanik) [inline Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java:26]: Not necessary

File: Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
+     */
+    public void testApp()
+    {
+        assertTrue( true );
```
Comment (npogulanik) [inline Maven_Topic2/marven-app/src/test/java/com/bootcamp/app/AppTest.java:36]: What are you testing here ?

**Comment by npogulanik** [comment]
Please read some docs on how to use junit with annotations. Is easy and simple.

**Comment by TecEliMoran** [comment]
This comments and source code are not mine, excep for the ona that asks for your name. It is all generatede by the comands in the console

**Pull request merged**